src/main/resources/application.properties

```diff
@@ -1 +1,13 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
+spring.datasource.username=${database_username}
+spring.datasource.password=${database_password}
+spring.datasource.driver-class-name=org.postgresql.Driver
+database_username =
+database_password =
+
+spring.jpa.database-platform=org.hibernate.dialect.PostgreSQL9Dialect
+spring.jpa.properties.hibernate.temp.use_jdbc_metadata_defaults = false
+
+spring.liquibase.change-log=classpath:/mavenPostgreSQL/dbchangelog.xml
```

Comment:
jak przeniesiesz dbchangelog.xml z mavenPostgreSQL do resource to ta linijka będzie do wywalenia